### ISSUE TITLE
[UX] Add a link to documentation in the "Congratulations, you installed Backdrop CMS" message.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1908]
This is the first thing that people see after installing Backdrop. Let's add a link to our user guide there that opens in a new window/tab.

**Comment by Bojhan** [comment]
Arguably you shouldn't congratulate on something that should be obvious and easy. 

I thought Backdrop purposely didnt push on documentation, thereby focusing on making the system easier vs. adding more links to documentation

**Comment by klonos** [comment]
No matter how easy a system is made in general, that will be relative to the knowledge of the user and some documentation will always be required.

**Comment by jenlampton** [comment]
@klonos this should be resolved now since we first show people the dashboard, and it contains a link to the user guide.

**Comment by klonos** [comment]
Agreed @jenlampton 👍 ...plus we also have an issue for rewording the "Congratulations!" bit: #4162

**Comment by stpaultim** [comment]
> this should be resolved now since we first show people the dashboard, and it contains a link to the user guide.

Actually, after installing Backdrop CMS a user is automatically logged in and taken directly to the front page, not the dashboard. This is confusing, because after logging in to an existing site, a user is taken to the dashboard.

**Comment by klonos** [comment]
Hmm 🤔 ...lets reopen this then.
